custom.css

```diff
@@ -1,3 +1,15 @@
+/* Center image via hashtag at end of img src */
+img[src*='#center'] {
```

Comment:
Sure @k4ml  ! That is called magic, nah..just kidding.. it is "attribute selectors"
https://developer.mozilla.org/en/docs/Web/CSS/Attribute_selectors